Source/WebKit/WebProcess/cocoa/WebProcessCocoa.mm

```diff
@@ -1503,6 +1503,13 @@ static float currentBacklightLevel()
 }
 #endif
 
+#if ENABLE(NOTIFYD_BLOCKING_IN_WEBCONTENT)
+void WebProcess::postNotification(const String& message)
+{
+    notify_post(message.utf8().data());
```

Comment:
Seems you could use `.ascii().data()`

Reply:
Thanks for reviewing, all!

I will address these issues in the upcoming patch :)